README.md

```diff
@@ -1,30 +1,45 @@
 # TalonOne - the C# library for the Talon.One API
 
 The Talon.One API is used to manage applications and campaigns, as well as to
-integrate with your application. The operations in the _Integration API_ section
-are used to integrate with our platform, while the other operations are
-used to manage applications and campaigns.
+integrate with your application.
```

Comment:
All these micro changes are very likely to be overridden next time we generate a version using the code SDK generator 🙈 

Not that I think we should revert them, just more of a mental comment to myself to try and remember to look at it as part of the next generation :)